scripts/build.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/bash -l
+# only execute these lines if the `module` command is present in the environment
+# used for the BU SCC
+if command -v module &>/dev/null; then
+    module load gcc/12.2.0
+    module load miniconda
+    conda env create -f "/projectnb/respond/environment.yml" 2&>/dev/null
+    conda activate respond
+fi
+
+# help message to be output either with the -h flag or when using invalid syntax
+showhelp () {
+    printf "\033[31m%s\033[0m" "$0 - Build the DataManagement Model"
```

Comment:
Do we consider DataManagement a model?

Reply:
We use to, and honestly I want to in the future. But for calibration we've temporarily removed it.